.github/workflows/bench.yml

```diff
@@ -4,7 +4,7 @@ on:
   workflow_dispatch:
   push:
     branches:
-      - main
```

Comment:
Is this intentional?

Reply:
I am just testing it out to see the benchmark results, will revert it once done

Reply:
reverted it, will test in an alternate branch